projects/ngx-charts-on-fhir/src/lib/fhir-mappers/component-observation-mapper.service.ts

```diff
@@ -1,10 +1,10 @@
 import { Injectable, Inject, forwardRef } from '@angular/core';
 import { ScaleOptions } from 'chart.js';
 import { Observation, ObservationComponent } from 'fhir/r2';
-import { fromPairs, merge } from 'lodash-es';
+import { merge } from 'lodash-es';
 import { DataLayer } from '../data-layer/data-layer';
 import { Mapper } from '../fhir-converter/multi-mapper.service';
-import { ChartAnnotation } from '../utils';
+import { ChartAnnotation, } from '../utils';
```

Comment:
@semicolin please remove extra comma

Reply:
Oops, I forgot to run prettier. I will create a second PR for code formatting.